lib/sail/value_cast.rb

```diff
@@ -50,6 +56,10 @@ def range_set(value)
       value.to_i
     end
 
+    def date_set(value)
+      Time.parse(value).strftime("%d-%m-%Y")
```

Comment:
Here, since the value will just be a date in string and we already have the validation in the model, we don't need any data modification. This would simply be

```ruby
def date_set(value)
  value
end
```

Reply:
@vinistock You are right, ill fix it. Not sure why I overlooked it.